Parsers/ASimProcessEvent/Parsers/vimProcessCreateMicrosoftSysmon.yaml

```diff
@@ -16,94 +16,69 @@ Description: |
-                        TargetUserSessionGuid = LogonGuid, 
-                        TargetProcessId = ProcessId,
-                        TargetProcessGuid = ProcessGuid,
-                        TargetProcessName = Image,
-                        TargetProcessIntegrityLevel = IntegrityLevel,    
-                        TargetProcessCompany = Company,
-                        TargetProcessFileDescription = Description,
-                        TargetProcessFileVersion = FileVersion,
-                        TargetProcessFileProduct = Product,   
-                        ActingProcessId = ParentProcessId,
-                        ActingProcessGuid = ParentProcessGuid,                          
-                        ActingProcessCommandLine = ParentCommandLine,
-                        ActingProcessName = ParentImage
-            | extend 
-                        TargetUsernameType = iff(isnotempty(TargetUsername),'Windows', ''),
-                        ActorUsernameType = iff(isnotempty(ActorUsername),'Windows', ''),
-                        // aliases
-                        User = TargetUsername,
-                        Process = TargetProcessName,
-                        Dvc = DvcHostname,
-                        Hash = coalesce(TargetProcessSHA256, TargetProcessSHA1, TargetProcessMD5) // which appears first - will be aliases to "Hash"
-            | project-away EventData, RuleName, ParameterXml
-            };
-    // this is the parser for sysmon from WindowsEvent table
-       let ParsedProcessEvent_WindowsEvent=(){
-        WindowsEvent
-        | where Provider == "Microsoft-Windows-Sysmon" and EventID==1
-        | extend hash = split(EventData.Hashes,'=')
-        | extend hash_algorithm = tostring(hash[0]),
-                    hash_value = tostring(hash[1])
-        | extend TargetProcessMD5 = iff(hash_algorithm == "MD5", hash_value, ""),
-                TargetProcessSHA1 =  iff(hash_algorithm == "SHA1", hash_value, ""),
-                TargetProcessSHA256 = iff(hash_algorithm == "SHA256", hash_value, ""),
-                TargetProcessIMPHASH = iff(hash_algorithm == "IMPHASH", hash_value, "")
-        | project-away hash, hash_algorithm
-        | extend 
+    // =======================================================================================================================================
+    //    Sysmon from Event table
+    // =======================================================================================================================================
+    let Sysmon11_Event=(){
+    Event
+    | where Source == "Microsoft-Windows-Sysmon" and EventID==1
+    // --------------------------------------------------------------------------------------
+    | where
+    (isnull(starttime)   or TimeGenerated >= starttime )
+    and (isnull(endtime) or TimeGenerated <= endtime )
+    and (disabled)
+    and (array_length(commandline_has_any)==0 or EventData has_any (commandline_has_any)) 
```

Comment:
all before any

Reply:
@oshezaf , made the changes. please review